calico_cni/policy_drivers.py

```diff
@@ -320,6 +307,21 @@ def _escape_chars(self, unescaped_string):
         return re.sub('[^a-zA-Z0-9\.\_\-]', swap_char, unescaped_string)
 
 
+class DefaultDenyInboundDriver(KubernetesAnnotationDriver):
```

Comment:
You'll need to override remove_profile as well I think. 

The annotation based driver tries to delete the profile on pod deletion which:
1) we don't want to do for this driver 
2) will output a warning whenever a pod is deleted.